media.css

```diff
@@ -0,0 +1,442 @@
+    flex-direction: column;
+    margin-right: 65px;
+    margin-bottom: 30px;
+  }
+
+  .task ul {
+    display: flex;
+    gap: 12px;
+    border-radius: 8px;
+    margin-right: 50px;
+    color: #05070a;
+    list-style-type: none;
+    text-align: start;
+    font-family: 'Poppins', sans-serif;
+    font-size: 18px;
+    line-height: 24px;
+    font-weight: 500;
+    margin-left: 43px;
+  }
+
+  .canopy {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .describe {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 75%;
+    width: 100%;
+
+    /* margin: 30px; */
+    row-gap: 10px;
+    padding-top: 142px;
+    padding-bottom: 555px;
+    margin-bottom: -516px;
+  }
+
+  h3 {
+    margin-left: -88px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    padding: 60px;
+    color: #344563;
+    margin-bottom: 16px;
+  }
+
+  .description {
+    margin-left: -30px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    margin-top: -50px;
+    max-width: 145%;
+    width: 145%;
+  }
+
+  .connect {
+    padding-bottom: 20px;
+    color: rgb(127, 140, 255);
+    margin-left: -14px;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .social-icon {
+    display: flex;
+    list-style-type: none;
+    margin-left: -100px;
+  }
+
+  .press {
+    margin-left: -22px;
+  }
+
+  .skill {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    margin-right: 142px;
+    margin-left: -10px;
+    margin-top: 120px;
+  }
+
+  .languages {
+    display: flex;
+    width: 85%;
+    justify-content: center;
+    flex-direction: row;
+    margin-left: -210px;
+  }
+
+  .options {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    padding: 10px;
+    column-gap: 115px;
+    margin-left: -30px;
+  }
+
+  .tech-img {
+    padding: 14px;
+    display: flex;
+    margin-left: -17px;
+    justify-content: flex-start;
+  }
+
+  .tech {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-left: -10px;
+    margin-top: 8px;
+    color: #253858;
+  }
+
+  .arw {
+    margin-left: 555px;
+  }
+
+  .arrow {
+    margin-right: 150px;
+    margin-left: 10px;
+  }
+
+  hr {
+    width: 82%;
+    margin-left: 0;
+  }
+
+  .abt-self {
+    /* margin-right: 250px; */
+    padding: 140px;
+    margin-bottom: -15px;
+  }
+
+  .shape {
+    display: none;
+  }
+
+  .background-shape {
+    display: none;
+  }
+
+  .contact-form {
+    margin-top: -120px;
+    margin-bottom: -100px;
+    background-image: url(assets/images/desktop\ contact.svg);
+  }
```

Comment:
- [OPTIONAL] Kindly make sure after making the desktop version of the project your mobile version is still rendering correctly. Please fix the margins of the contact form section. :pray:  :pray: 
Your Design:
![image](https://user-images.githubusercontent.com/98363075/193422088-137fb9f2-d3c6-4aa6-b7b3-358325613328.png)

Figma Design:
![image](https://user-images.githubusercontent.com/98363075/193422136-ef17b7d4-0e6f-4d43-9882-c3eacaf70bd8.png)